### PR TITLE
add UT for cluster/images/etcd/migrate/copy_file.go

### PR DESCRIPTION
**Pull request by liyuerich**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:

add UT for cluster/images/etcd/migrate/copy_file.go

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?
None

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

None

### CODE REVIEW

File: cluster/images/etcd/migrate/copy_file_test.go [hunk 1]
```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
```
Comment (charles-chenzz) [inline cluster/images/etcd/migrate/copy_file_test.go:3]: 2023

File: cluster/images/etcd/migrate/copy_file_test.go [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+		t.Errorf("file content is not the same")
+	}
+
+	// clean up testing file
+	os.Remove("test2.txt")
+}
+
+func createTestFile(s1 string) (*os.File, error) {
+	return os.Create(s1)
+}
```
Comment (charles-chenzz) [inline cluster/images/etcd/migrate/copy_file_test.go:61]: any other places that use this function? if not I don't think we need it here, we can just call os.Create() in line31

**Comment by liyuerich** [comment]
thanks charles, both are resolved. Please review again.

**Comment by jiahuif** [comment]
/assign @charles-chenzz 
I am not sure why we need a test of copying files. Do you have the context? Thank you.
/triage accepted

**Comment by charles-chenzz** [comment]
> I am not sure why we need a test of copying files. Do you have the context? Thank you.

I have no context here as well, but imo:
pros: improve the test coverage
cons: we might introduce some useless test into k/k (we can guarantee the code will absolutely working)

**Comment by liyuerich** [comment]
as copying files is very basic method, and do not need to run the unit test. I suppose I could close this PR.  thanks all for your   very helpful comments.

**Pull request closed without merging**